lightning/src/chain/keysinterface.rs

```diff
@@ -319,13 +319,13 @@ pub trait ChannelKeys : Send+Clone {
 	/// protocol.
 	fn sign_channel_announcement<T: secp256k1::Signing>(&self, msg: &UnsignedChannelAnnouncement, secp_ctx: &Secp256k1<T>) -> Result<Signature, ()>;
 
-	/// Set the remote channel basepoints and remote/local to_self_delay.
+	/// Set the remote channel basepoints and counterparty_selected/locally_selected_contest_delay.
```

Comment:
Should we change the function names in ChannelKeys (above)

Reply:
Add a new commit scoping ChannelKeys under new nomenclature.